docs/self-hosted-appcircle/configure-server/_firewalld-configuration.mdx

```diff
@@ -0,0 +1,23 @@
+If you are using `Firewalld`, you need to open the ports below according to your server configuration.
+
+- If you plan to run the Appcircle server with HTTPS:
+
+```bash
+sudo firewall-cmd --add-port=80/tcp --permanent
```

Comment:
Why do we need to open port `80` for HTTPS configuration?

Reply:
As explained below, we have a valid use case for the `80` port.
> ... _kullanıcılar tarayıcıya [store.spacetech.com](http://store.spacetech.com/) yazıp enterladıklarında, tarayıcı default olarak HTTP ve 80 portuna istek atıyor. Bu yönlendirme olmazsa (80->443) kullanıcı connection hatası alacaktır. Ancak yönlendirme sayesinde bunun da önüne geçilmiş oluyor._

So, we will add the reason here to the documents as an information box to prevent possible security questions.

cc/ @burakberkkeskin